SalaryService.Api/ci/values-custom.yaml

```diff
@@ -0,0 +1,227 @@
+## @section ASP.NET Core parameters
+##
+
+## Bitnami ASP.NET Core image
+## ref: https://hub.docker.com/r/bitnami/aspnet-core/tags/
+## @param image.registry ASP.NET Core image registry
+## @param image.repository ASP.NET Core image repository
+## @param image.tag ASP.NET Core image tag (immutable tags are recommended)
+## @param image.digest ASP.NET Core image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+## @param image.pullPolicy ASP.NET Core image pull policy
+## @param image.pullSecrets ASP.NET Core image pull secrets
+## @param image.debug Enable image debug mode
+##
+image:
+  registry: ghcr.io
+  repository: tourmalinecore/inner-circle/dev-salary-service-api
+  tag: latest
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+  ##
+  pullPolicy: Always
+
+  pullSecrets: []
+    # https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_images.tpl#L49
+    # - "{{ include \"aspnet-core.fullname\" . }}-docker-secret"
+
+  pullCredentials: {}
+    # username: login
+    # password: password
+
+  debug: false
+
+# Here you need to pass your application dll to the dotnet utility
+args:
+  - "SalaryService.Api.dll"
+  # - "EmailSender.Api.dll"
+
+## @param bindURLs URLs to bind
+##
+bindURLs: http://+:80
+
+# Here stored regular vars that passed to the pod environment
+extraConfigMapEnvVars:
```

Comment:
Could we have here default value for dictionary

Reply:
Added default value for this variable